bridge/sshchat/sshchat.go

```diff
@@ -31,8 +31,7 @@ func (b *Bsshchat) Connect() error {
 			b.w = w
 			b.r.Scan()
 			w.Write([]byte("/theme mono\r\n"))
-			b.handleSSHChat()
-			return nil
+			return b.handleSSHChat()
```

Comment:
I will revisit this function in a wider manner in a subsequent PR as the synchronisation logic in it is broken.